config/locales/models/en.yml

```
@@ -155,6 +155,13 @@ en:
               not_work: Only a link to a work can be listed as an inspiration.
               protected: You can't use the related works function to cite works by the protected user %{login}.
           format: "%{message}"
+        skin:
+          archive_in_title: You can't use the word 'Archive' in your skin title, sorry! (We have to reserve it for official skins.)
```

I talked with AD&T and Support about updating the wording for some of these, since it's been a bit and they don't quite match how we word things today. First suggestion:
```suggestion
          archive_in_title: Sorry, titles including the word 'Archive' are reserved for official skins.
```